ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/useFormikScanConfig.tsx

```diff
@@ -33,7 +33,13 @@ export const defaultScanConfigFormValues: ScanConfigFormValues = {
 
 const validationSchema = yup.object().shape({
     parameters: yup.object().shape({
-        name: yup.string().required('Scan name is required'),
```

Comment:
This isn't specific to validation, but I did notice that the UI didn't sort configurations by name in an ascending or descending order.

`compliance-enhanced/cluster-compliance/scan-configs?sortOption[field]=Name&sortOption[direction]=desc` and `compliance-enhanced/cluster-compliance/scan-configs?sortOption[field]=Name&sortOption[direction]=asc` present the same order with two separate scan configs.

Reply:
I'll check that, but if it's not a regression in this PR, I'll just file it a bug, to keep the changes on topic.